server/src/entity/Account.ts

```diff
@@ -46,6 +48,15 @@ export class Account extends BaseEntity {
   })
   public isActive: boolean;
 
-  @OneToMany((type) => Shift, (shift) => shift.server)
+  @OneToMany(() => Shift, (shift) => shift.server)
```

Comment:
Invalid syntax `@ManyToOne(type => User, user => user.photos)`

Reply:
What do you mean by invalid syntax... it works

Reply:
http://typeorm.io/#/many-to-one-one-to-many-relations